videoproviders/api/videofront.py

```diff
@@ -1,4 +1,5 @@
 from datetime import datetime
+from simplejson import JSONDecodeError
```

Comment:
The documentation refers to `requests.exceptions.JSONDecodeError`, but I assume this is due to the older version of requests that you mentioned

Reply:
it's even still in dev apparently https://github.com/psf/requests/blob/master/HISTORY.md

What's sure is that in the edxapp container:
```python
>>> from requests.exceptions import JSONDecodeError
Traceback (most recent call last):
  File "<console>", line 1, in <module>
ImportError: cannot import name JSONDecodeError
```

Reply:
ok :+1: